libs/hdf-converters/src/mappings/ChecklistMappingData.ts

```diff
@@ -0,0 +1,241 @@
+import _ from 'lodash';
+
+export type ChecklistFile = {
+  /**
+   * Unique identifier for this file. Used to encode which file is currently selected, etc.
+   *
+   * Note that in general one can assume that if a file A is loaded AFTER a file B, then
+   * A.unique_id > B.unique_id.
+   * Using this property, one might order files by order in which they were added.
+   */
+  uniqueId: string;
+  /** The filename that this file was uploaded under. */
+  filename: string;
+
+  database_id?: string;
+
+  asset: ChecklistAsset;
+  stigs: ChecklistStig[];
+  raw: Record<string, unknown>;
+};
+
+type ChecklistAsset = {
+  role: string;
+  assettype: string;
+  hostname: string;
+  hostip: string;
+  hostmac: string;
+  hostfqdn: string;
+  marking: string;
+  targetcomment: string;
+  techarea: string;
+  targetkey: string;
+  webordatabase: boolean;
+  webdbsite: string;
+  webdbinstance: string;
+};
+
+export type ChecklistStig = {
+  header: StigHeader;
+  vulns: ChecklistVuln[];
+};
+
+type StigHeader = {
+  version: string;
+  classification: string;
+  customname?: string;
+  stigid: string;
+  description: string;
+  filename: string;
+  releaseinfo?: string;
+  title: string;
+  uuid: string;
+  notice?: string;
+  source?: string;
+};
+
+type ChecklistVuln = {
```

Comment:
I have documented each field to where it is displayed (if at all) in StigViewer to help with typing and mapping to hdf. Not sure where this would be helpful to store yet for public viewing.